core/amber/src/main/python/texera_run_python_worker.py

```diff
@@ -22,8 +23,28 @@ def init_loguru_logger(stream_log_level) -> None:
 
 
 if __name__ == "__main__":
-    _, worker_id, output_port, logger_level, r_path = sys.argv
+    (
+        _,
+        worker_id,
+        output_port,
+        logger_level,
+        r_path,
+        iceberg_postgres_catalog_uri_without_scheme,
+        iceberg_postgres_catalog_username,
+        iceberg_postgres_catalog_password,
+        iceberg_table_namespace,
+        iceberg_file_storage_directory_path,
+        iceberg_table_commit_batch_size,
```

Comment:
I see it is indeed a lot of configurations to pass. Maybe consider using a JSON format string to pass a JSON object for hierarchy.

Reply:
Currently it is manageable. If more arguments are needed to be passed I will add a separate PR to use JSON.

Reply:
sure